test/test_filepost.py

```diff
@@ -2,9 +2,9 @@
 
 from urllib3.fields import RequestField
 from urllib3.filepost import encode_multipart_formdata, iter_fields
-from urllib3.packages.six import b
 
 BOUNDARY = "!! test boundary !!"
+LATIN_ENCODED_BOUNDARY = BOUNDARY.encode("latin-1")
```

Comment:
Instead of `LATIN_ENCODED_BOUNDARY` maybe `BOUNDARY_BYTES` is a more descriptive name?